cmd/integration_test/build/buildspecs/snow-test-eks-a-cli.yml

```diff
@@ -45,15 +49,16 @@ phases:
         -i ${INTEGRATION_TEST_INSTANCE_PROFILE}
         -m ${INTEGRATION_TEST_MAX_EC2_COUNT}
         -p ${INTEGRATION_TEST_MAX_CONCURRENT_TEST_COUNT}
-        -r 'Test'
```

Comment:
My vote is to do this filtering/skipping non-simpleflow tests through the SKIPPED_TESTS.yaml file while still keeping this regex as `Test`.

Reply:
changed